<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-06T20:29:01 CDT" name="Default suite" finished-at="2024-06-06T20:29:06 CDT" duration-ms="5514">
    <groups>
    </groups>
    <test started-at="2024-06-06T20:29:01 CDT" name="Default test" finished-at="2024-06-06T20:29:06 CDT" duration-ms="5514">
      <class name="tests.SFDCLogin">
        <test-method is-config="true" signature="doConfigReport()[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:00 CDT" name="doConfigReport" finished-at="2024-06-06T20:29:01 CDT" duration-ms="472" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doConfigReport -->
        <test-method is-config="true" signature="preConditions(java.lang.reflect.Method)[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:01 CDT" name="preConditions" finished-at="2024-06-06T20:29:05 CDT" duration-ms="4496" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.SFDCLogin.ValidateLoginErrorMessage_TC4B() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preConditions -->
        <test-method signature="ValidateLoginErrorMessage_TC4B()[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:05 CDT" name="ValidateLoginErrorMessage_TC4B" finished-at="2024-06-06T20:29:06 CDT" duration-ms="514" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.remote.DriverCommand.SET_IMPLICIT_WAIT_TIMEOUT(DriverCommand.java:354)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteTimeouts.implicitlyWait(RemoteWebDriver.java:933)
at tests.SFDCLogin.ValidateLoginErrorMessage_TC4B(SFDCLogin.java:145)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateLoginErrorMessage_TC4B -->
        <test-method is-config="true" signature="postConditions(java.lang.reflect.Method)[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:06 CDT" name="postConditions" finished-at="2024-06-06T20:29:06 CDT" duration-ms="223" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.SFDCLogin.ValidateLoginErrorMessage_TC4B() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postConditions -->
        <test-method is-config="true" signature="cleanup()[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:06 CDT" name="cleanup" finished-at="2024-06-06T20:29:06 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
        <test-method is-config="true" signature="tearDownConfigReport()[pri:0, instance:tests.SFDCLogin@2fab4aff]" started-at="2024-06-06T20:29:06 CDT" name="tearDownConfigReport" finished-at="2024-06-06T20:29:07 CDT" duration-ms="407" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownConfigReport -->
      </class> <!-- tests.SFDCLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
